### ISSUE TITLE
Build scripts don't generate Kurdish (ku) dll resource

### ISSUE THREAD

**Issue by ohmineer** [Humanizr/Humanizer#1160]
Hi,

I am trying to pass tests to submit a PR. Following the comment found in issue #1156 [(1156#issuecomment-1001682012)](https://github.com/Humanizr/Humanizer/issues/1156#issuecomment-1001682012), I can't either pass Kurdish tests when I run build.ps1 or build.cmd scripts. On the contrary, if I run the tests within Visual Studio's Test Explorer, everything turns green.

![Test Explorer](https://user-images.githubusercontent.com/8396492/148372062-bfb63f2d-c9a0-45fc-a37d-338ef486840e.png)
![Test Explorer Localisation "ku" Tests](https://user-images.githubusercontent.com/8396492/148372363-a9286cf3-6e38-4f0a-bef1-c3a44c6ab662.png)

I have noticed that "ku" folder in output directory does not exists or is empty when projects are restored upon running scripts. 
What happens then is that translation requested to ResourceManager default to English language in runtime.

![Test error when running scripts](https://user-images.githubusercontent.com/8396492/148372579-eaaa9967-5830-4647-9961-2dfc0e3d4e37.png)

This folder contains the corresponding dll when running tests in Visual Studio but I can't seem to understand the reason as there is no difference among "ku" resource file and the rest of localised files. Is anybody able to help with this?

Just in case it helps, dotnet --info produces this in my machine:
![image](https://user-images.githubusercontent.com/8396492/148373921-6f4feb30-ce49-406a-87cb-df43d8db4f2e.png)

Thanks in advance,

**Comment by ohmineer** [comment]
I figured out my problem. I installed Central Kurdish language in my system and now the script is executed successfully.

![Green tests!!!](https://user-images.githubusercontent.com/8396492/148419520-8a6e80c0-c9f1-4400-9aa5-30e279b240d3.png)

My system is configured in Spanish and no other languages appear in Windows configuration.
For that reason, I am not sure why this action is needed for this specific language and not for others (Asian languages)

![Language configuration in Windows 10](https://user-images.githubusercontent.com/8396492/148420062-c4b6002d-3f79-41a6-8a32-ca7861eda2fe.png)

If any other user can confirm this is the solution, I would be happy to close this issue.

**Comment by twwilliams** [comment]
I don't think that having to install the Kurdish language pack in our development machines is a reasonable requirement, especially since it isn't needed for other languages.

Something must be different about the way the .ku localization has been implemented compared to all the other languages.

**Comment by ohmineer** [comment]
I understand your concerns. I suggested you try this while someone with more experience with the project can provide any insights on what could be happening. That feedback might help narrowing down the issue.

On the other hand, I reviewed the implementation of "ku" localisation. I can't see anything different. Just in case you can help, everything seems to start with this commit 2ba67c0e9fd15e1ade21819a02744960010e5131.

One final comment, I noticed a few Arabic fonts were installed in my system when I installed the language pack. You may want to try that as it looks less "harmful"...

![Arabic fonts](https://user-images.githubusercontent.com/8396492/148522014-7abd71e8-7afa-454b-ae6e-ea3e090c735c.png)

I hope it helps...

**Comment by neilboyd** [comment]
See also #1177 and #1180

**Comment by SimonCropp** [comment]
also related https://github.com/dotnet/msbuild/issues/1454

**Comment by clairernovotny** [comment]
Thank you for the detailed report and investigation. Localization is now generated from checked-in YAML and shipped in the main package, so the old build-time Kurdish satellite DLL path no longer exists. This structural fix is on main and has not yet been included in a numbered release.
